### PR TITLE
Password Hash and Token Generation added

### PR DESCRIPTION
**Pull request by arnabsen1729**
- Password is hashed using `bcrypt` before storing in db
- Login verification added

### CODE REVIEW

**Comment by arnabsen1729** [comment]
@raydwaipayan  do check it out

File: docker-compose.yml [hunk 1]
```diff
@@ -6,26 +6,12 @@ services:
...
-    container_name: pgadmin_container
-    image: dpage/pgadmin4
-    environment:
-      PGADMIN_DEFAULT_EMAIL: ${PGADMIN_EMAIL}
-      PGADMIN_DEFAULT_PASSWORD: ${PGADMIN_PASSWD}
-    volumes:
-       - ./pgadmin:/root/.pgadmin
-    ports:
-      - "5050:80"
+       - "127.0.0.1:5432:${DB_PORT}"
```
Comment (raydwaipayan) [inline docker-compose.yml]: This is wrong.
In a bind, port1:port2,
port1 in your local system will be bound to port2 in
docker's internal network stack.

File: server/handler/user_handler.go [hunk 2]
```diff
@@ -17,7 +19,17 @@ func Register(c *fiber.Ctx) error {
 	if err := c.BodyParser(u); err != nil {
 		return c.SendStatus(fiber.StatusBadRequest)
 	}
-	u.Create(models.DBConfigURL)
+	pass := []byte(u.Password)
+	hashedPassword, err := bcrypt.GenerateFromPassword(pass, bcrypt.DefaultCost)
+	if err != nil {
+		panic(err)
```
Comment (raydwaipayan) [inline server/handler/user_handler.go]: Don't panic in production.
Use log.Fatal(err) instead

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 pgadmin/
 postgres/
 .env
+bin/
```
Comment (raydwaipayan) [inline .gitignore]: missing newline at end of file

**Review by raydwaipayan — CHANGES_REQUESTED** [review]

**Review by raydwaipayan — APPROVED** [review]
LGTM

**Pull request merged**